Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+#!groovy
```

Comment:
There is no decision tree, declarative pipelines can include script blocks which container scripted groovy. It just discourages writing too much in scripts. This is not a blocker btw.